openapi/src/parts/controlled_azure_storage_container.yaml

```diff
@@ -24,6 +24,28 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/workspaces/v1/{workspaceId}/resources/controlled/azure/storageContainer/{resourceId}:
+    post:
+      parameters:
+        - $ref: '#/components/parameters/WorkspaceId'
+        - $ref: '#/components/parameters/ResourceId'
+      summary: Delete an Azure Storge Container
+      operationId: deleteAzureStorageContainer
+      tags: [ControlledAzureResource]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DeleteControlledAzureResourceRequest'
+      responses:
+        '200':
+          $ref: '#/components/responses/DeleteControlledAzureResourceResponse'
+        '403':
```

Comment:
Should we document the 404 when resource or workspace isn't found? I see that 404 isn't documented in other similar cases, so maybe there's a reason to omit it

Reply:
We do not return 404 if the target of the delete is not found. That is on purpose; it helps with idempotency.
However, if the containing workspace is not found, I would expect we would get a 404. There probably should be documented return and it is probably incorrect in the other similar cases.

Reply:
Thanks both. I've added a 404 response for this endpoint, ticketed up the remediation for the rest as [WOR-490](https://broadworkbench.atlassian.net/browse/WOR-490)